cogboard-webapp/src/components/PopoverWithControls.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import copy from 'copy-to-clipboard';
+import { CaptionWithPointer, StyledPopoverText } from './styled';
+import { Button, Popover } from '@material-ui/core';
+
+export const PopoverWithControls = ({ title, body, withCopy }) => {
```

Comment:
This is a bit of a copy-paste from `ServiceCheckWidget`, aligned to fit what I needed in here